### ISSUE TITLE
Internal sever errors on event query

### ISSUE THREAD

**Issue by berndbohmeier** [trustlines-network/relay#81]
Queries like 
`relay0.testnet.trustlines.network:443/api/v1/users/0x5b83F2f81186c5908C57E70F9c68B9F934f1d048/events` result in internal server error 500. 

```
Traceback (most recent call last):
relay_1    |   File "/usr/local/lib/python3.5/site-packages/gevent/greenlet.py", line 534, in run
relay_1    |     result = self._run(*self.args, **self.kwargs)
relay_1    |   File "/usr/local/lib/python3.5/site-packages/relay/blockchain/currency_network_proxy.py", line 162, in get_network_events
relay_1    |     result = list(itertools.chain.from_iterable([event.value for event in events]))
relay_1    | TypeError: 'NoneType' object is not iterable
relay_1    | <Greenlet at 0x7f2b682ece88: <bound method CurrencyNetworkProxy.get_network_events of <relay.blockchain.currency_network_proxy.CurrencyNetworkProxy object at 0x7f2b699fcc50>>('Transfer', from_block=396, user_address='0x5b83F2f81186c5908C57E70F9c68B9F934f1d048')> failed with TypeError
```

**Comment by schmir** [comment]
I've accidentally fixed that in c033a2e2bedcb930317d2039f7d846ff012accc4. Sorry.

**Comment by berndbohmeier** [comment]
Fixed
